server_addon/max/client/ayon_max/api/lib.py

```diff
@@ -573,3 +607,17 @@ def suspended_refresh():
     finally:
         rt.enableSceneRedraw()
         rt.resumeEditing()
+
+
+@contextlib.contextmanager
+def scene_fps(fps):
+    """Set scene fps during context
+
+    Args:
+        fps (float): fps value
+    """
+    try:
+        rt.frameRate = float(get_fps_for_current_context())
+        yield
+    finally:
+        rt.frameRate = fps
```

Comment:
Logic here is wrong.

Should be something like:

```suggestion
    fps_before = rt.frameRate
    try:
        rt.frameRate = float(fps)
        yield
    finally:
        rt.frameRate = fps_before
```